admin/src/scenes/classe/components/modale/modaleCohort.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { HiOutlineExclamation } from "react-icons/hi";
+import { ModalConfirmation } from "@snu/ds/admin";
+
+interface Props {
+  modaleCohort: boolean;
+  setModaleCohort: (b: boolean) => void;
```

Comment:
```suggestion
  isOpen?: boolean;
  onClose: () => void;
```
le boolean n'est pas nécessaire dans le `onClose`, on ne demande jamais l'ouverture de la modal

Reply:
onClose={() => {
        onClose(false);
      }}
      j'ai une erreur de type ici si je ne le met pas